Index: src/homework/fileUtil/FileUtil.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/homework/fileUtil/FileUtil.java	(date 1678975499000)
+++ src/homework/fileUtil/FileUtil.java	(date 1678975594176)
@@ -90,7 +90,6 @@
         }catch (Exception e){
             System.out.println("wrong address entered");
         }
-
     }
 
     //այս մեթոդը պետք է սքաններով վերցնի մեկ string.
